requirements.txt

```diff
@@ -1,51 +1,28 @@
 # keyring is required to provide GitHub token to EasyBuild;
 # for recent versions of keyring, keyrings.alt must be installed too
-keyring; python_version >= '2.7'
-keyrings.alt; python_version >= '2.7'
+keyring
+keyrings.alt
 
-# GitDB 4.0.1 no longer supports Python 2.6
-gitdb==0.6.4; python_version < '2.7'
-gitdb; python_version >= '2.7'
+# GitDB
+gitdb
 
 # GitPython 3.1.15 deprecates Python 3.5
 GitPython==3.1.14; python_version >= '3.0' and python_version < '3.6'
-# GitPython 2.1.9 no longer supports Python 2.6
-GitPython==2.1.8; python_version < '2.7'
-GitPython; python_version >= '3.6' or (python_version <= '3.0' and python_version >= '2.7')
+GitPython; python_version >= '3.6' or python_version <= '3.0'
 
+# pydot
+pydot
```

Comment:
@branfosj This can be removed, we only had this as a separate entry to control which `pydot` version is used as dependency for `python-graph-dot`

Reply:
done in a920e85